ec2.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_security_group" "main" {
+  name        = "main"  
+  description = "Allow SSH traffic"
+  # allow ingress ("in") traffic on port 22, the port SSH runs on
+  ingress {
```

Comment:
Leaving this as a placeholder comment (I'm not requesting any change here):

Having this ingress rule is convenient for us to be able to immediately SSH to the newly created EC2 instance, but if this EC2 instance was for a _real_ application to run on, we would _NOT_ want full SSH access from any IP since that would be a major security risk.

Reply:
Actually, let's remove this rule and just add a MyIP rule for port 22 any time we create a new server.